### PR TITLE
create sign-in component

### LINKED ISSUE #36: [Frontend] - Create the sign-in component
## Description

As a developer
With regards to the screenshot below and the [figma file](https://www.figma.com/file/UIPnjph81ynxLM3IUUdM8R/Spaceyatech-design-system?node-id=472%3A5732) 
Assets on figma file 

## Screenshot 
image

## Reference
- [Figma design](https://www.figma.com/file/UIPnjph81ynxLM3IUUdM8R/Spaceyatech-design-system?node-id=472%3A5732)

### PR DESCRIPTION
**Pull request by NikolaGundzikova**
# Description

Sign in component created.

Fixes #36 

## New component with tsx and css

No tests - no functions.

# Checklist:

- [ ✓ ] My code follows the style guidelines of this project
- [ ✓ ] I have performed a self-review of my own code
- [ ✓ ] My changes generate no new warnings
- [ ✓ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by nyandika — COMMENTED** [review]
Include the entire .idea folder in gitignore

**Comment by NikolaGundzikova** [comment]
> Include the entire .idea folder in gitignore

thank you, solved

File: src/components/sign-in-component.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            <div className="rightSide">
+                <div className="buttonsBody">
+                    <h1 className="firstText"> Login to Space Ya tech</h1>
+
+                    <div>
+                        <form className="inputsP">
+                            <div className="username">
+                                <label>
+                                    <input className="sign-inInputs" type="text" name="name"
+                                           placeholder="Username Or Email adress"/>
```
Comment (nyandika) [inline src/components/sign-in-component.tsx]: nit: typo

**Review by nyandika — APPROVED** [review]
Looks good

File: src/components/sign-in-component.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React from "react";
```
Comment (daveclinton) [inline src/components/sign-in-component.tsx]: Use this instead `import * as React from 'react';`

File: src/components/sign-in-component.css [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+}
+
+.horizontalLine {
+    position: relative;
+    display: flex;
+    padding-bottom: 24px;
+    padding-top: 44px;
+}
+
+.OR {
```
Comment (daveclinton) [inline src/components/sign-in-component.css]: Add descrptive className here

File: src/components/sign-in-component.css [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+    align-items: center;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 32px;
+
+}
+
+.loginFacebook {
```
Comment (daveclinton) [inline src/components/sign-in-component.css]: All the icons can use the same styling to reduce lines of code

**Review by daveclinton — CHANGES_REQUESTED** [review]

**Review by daveclinton — CHANGES_REQUESTED** [review]
Use branch naming convention from contribution guidelines of this repository. Also check commit message guidelines

**Review by daveclinton — CHANGES_REQUESTED** [review]
Use camelCase for naming files. i.e signInComponent

**Review by daveclinton — CHANGES_REQUESTED** [review]
Use camelCase for naming files. i.e signInComponent.
And why do you have components inside if the assets folder?

**Comment by NikolaGundzikova** [comment]
Hi, I updated PR.

**Comment by daveclinton** [comment]
> Hi, I updated PR.

Hey @NikolaGundzikova Thats good. You can do a `git pull origin --rebase pre-dev` to fix the conflicts you have

**Review by jumalaw98 — CHANGES_REQUESTED** [review]
Check out the conflict and try, resolving,

**Comment by NikolaGundzikova** [comment]
I resolved conflicts.

**Comment by jumalaw98** [comment]
@NikolaGundzikova  check out on this, still have some minor conflicts to tackle
image

**Comment by NikolaGundzikova** [comment]
> @NikolaGundzikova check out on this, still have some minor conflicts to tackle image

I don´t know why is merging blocked because I updated my branch. Everything looks good.

**Review by jumalaw98 — APPROVED** [review]

**Comment by NikolaGundzikova** [comment]
> > Hi, I updated PR.
> 
> Hey @NikolaGundzikova Thats good. You can do a `git pull origin --rebase pre-dev` to fix the conflicts you have

Can you check my change, please?

**Pull request merged**